paper-textarea.html

```diff
@@ -131,15 +131,19 @@
     },
 
     _ariaLabelledByChanged: function(ariaLabelledBy) {
-      this.$.input.textarea.setAttribute('aria-labelledby', ariaLabelledBy);
+      this.textareaElement.setAttribute('aria-labelledby', ariaLabelledBy);
     },
 
     _ariaDescribedByChanged: function(ariaDescribedBy) {
-      this.$.input.textarea.setAttribute('aria-describedby', ariaDescribedBy);
+      this.textareaElement.setAttribute('aria-describedby', ariaDescribedBy);
     },
 
     get _focusableElement() {
-      return this.$.input.textarea;
+      return this.textareaElement;
     },
+
+    get textareaElement() {
```

Comment:
This shooooould be ok if we populate `this.$` correctly, right?

Reply:
Sadly this is actually the biggest issue I have so far. This is the main reason the tests fail ... the `paper-textarea`.

Reply:
Do you have any more details as to what fails? Are you setting the this.$ correctly?
I can check this out locally if you want